src/components/ApplicationCard.tsx

```diff
@@ -0,0 +1,259 @@
+  const [needsQ5, setNeedsQ5] = useState("");
+
+  const maxTextArea = 500;
+
+  useEffect(() => {
+    async function preloadFields() {
+      const project = await getNonprofitProject(projectId);
+      const nonprofit = project.nonprofit as Nonprofit;
+
+      setProjectName(project.name);
+      setIsVerified(nonprofit.isVerified);
+
+      if (isRead) {
+        const application = await getApplication(projectId);
+
+        setAboutQ1(application.aboutQ1 ?? "");
+        setAboutQ2(application.aboutQ2 ?? "");
+        setAboutQ3(application.aboutQ3 ?? "");
+        setAboutQ4(application.aboutQ4 ?? "");
+        setNeedsQ1(application.needsQ1 ?? "");
+        setNeedsQ2(application.needsQ2 ?? "");
+        setNeedsQ3(application.needsQ3 ?? "");
+        setNeedsQ4(application.needsQ4 ?? "");
+        setNeedsQ5(application.needsQ5 ?? "");
+      }
+    }
+
+    preloadFields().catch((error: Error) => {
+      showError(error.message);
+    });
+  }, []);
+
+  const handleSubmit = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    const applicationCreate: NonprofitApplicationCreate = {
+      aboutQ1: aboutQ1 !== "" ? aboutQ1 : undefined,
+      aboutQ2: aboutQ2 !== "" ? aboutQ2 : undefined,
+      aboutQ3: aboutQ3 !== "" ? aboutQ3 : undefined,
+      aboutQ4: aboutQ4 !== "" ? aboutQ4 : undefined,
+      needsQ1: needsQ1 !== "" ? needsQ1 : undefined,
+      needsQ2: needsQ2 !== "" ? needsQ2 : undefined,
+      needsQ3: needsQ3 !== "" ? needsQ3 : undefined,
+      needsQ4: needsQ4 !== "" ? needsQ4 : undefined,
+      needsQ5: needsQ5 !== "" ? needsQ5 : undefined,
+    };
+
+    try {
+      await createNonprofitApplication(projectId, applicationCreate);
+      showInfo("Successfully submitted application.");
+    } catch (e) {
+      const error = e as Error;
+      showError(error.message);
+    }
+  };
+
+  return (
+    <VStack
+      maxW="1000px"
+      border="1px solid #BCC5D1"
+      borderRadius={10}
+      backgroundColor="surface"
+      spacing={3}
+      align="stretch"
+    >
+      <Box
+        borderRadius="10px 10px 0px 0px"
+        border="1px solid #BCC5D1"
+        backgroundColor="primary"
+        p={3}
+      >
+        <Text color="white" fontSize="md" fontWeight={700} marginLeft={7}>
+          {projectName}
+        </Text>
+      </Box>
+      <VStack align="stretch" p={12} spacing={12}>
+        {!isRead && !isVerified && (
+          <VStack align="start" spacing={5}>
+            <Text alignSelf="flex-start" fontSize="md" fontWeight={700}>
+              Nonprofit Verification
+            </Text>
+            <FormControl isReadOnly={isRead}>
+              <FormLabel fontSize="sm">Question 1</FormLabel>
+              <VStack align="flex-start" spacing={4}>
+                <FormHelperText fontSize="sm">
+                  Enter your EIN so we can verify you as a nonprofit.
+                </FormHelperText>
+                <Input
+                  pattern="^[0-9]{2}-[0-9]{7}$"
```

Comment:
Using pattern will not be useful here because it is not within the context of a form.  You could either wrap the application with the form tag and use the onSubmit handler, or you could compare the ein to this pattern when the application is being submitted.  If you go with the latter (which should be easier), go ahead and remove this pattern prop and use your QuestionCard here as well.